From 9c31b78275edc8bba9e91cd073c13688c8d7cd3b Mon Sep 17 00:00:00 2001
From: Tao <tao@tukeq.com>
Date: Fri, 13 Jan 2012 17:07:54 +0800
Subject: [PATCH 5/5] templatetags bugfix

---
 pipeline/templatetags/compressed.py |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/pipeline/templatetags/compressed.py b/pipeline/templatetags/compressed.py
index 4ab73d5..fc17d48 100644
--- a/pipeline/templatetags/compressed.py
+++ b/pipeline/templatetags/compressed.py
@@ -44,6 +44,10 @@ class CompressedCSSNode(template.Node):
             context.update({
                 'rel': 'stylesheet/less'
             })
+        else:
+            context.update({
+                'rel': 'stylesheet'
+            })
         return render_to_string(package['template'], context)
 
     def render_individual(self, package):
-- 
1.7.7.3

